blog/2021-09-15-soc2-things-i-learned-part-2.md

```diff
@@ -0,0 +1,147 @@
+
+With Martin's help we were able to utilize the same project management tools that we already use to track our compliance efforts.
+
+As it turns out, this was a pretty big deal.
+
+This is because everyone in the company has access to it.
+Everyone can see the control statements, what criteria those statements map to, the policies that we wrote for processes to adhere to, the progress of any given compliance effort, etc...
+It also means that we can easily see how any given internal task rolls up under a compliance related effort, or has been generated by it.
+Essentially, it puts the SOC 2 process right at the heart of our operating mind set, which is so critical for the long term.
+
+### 4. Use a reliable auditing partner
+
+There are many companies out there that offer this service.
+The truth is, finding the right one for your company can be a little hit or miss.
+That is because, each company does things a little differently.
+
+We have worked with two different ones so far, and for us we found our experience with [Schneider Downs](https://www.schneiderdowns.com/) to be really good.
+Martin recommended them, and has worked with them for a few years.
+
+They pros that we really liked were:
+- They were highly communicative.
+- They had previous experience with developer facing SaaS companies.
+- They had a polished system for evidence submission that gave us clear visibility into the status of our audit.
+
+I definitely can not say this is the case for the other companies we talked to our worked with previously.
+
+### 5. Spread out the workload
+
+While spreading out the workload helps to prevent any single person from getting overwhelmed, it is also about knowing the roles that individuals play in the compliance effort.
+For us, we found that any given person generally falls into one of three roles.
+
+#### The Department Head
+
+This is a very important role.
+The Department Head's responsibility is to make sure that people are doing the things we say we are doing.
+The Department Head must have a good sense of control statements that fall into their area.
+They are the "buck stops here" in terms of driving their department to think and care about the compliance process.
+
+In our last period we had four Department Heads who were heavily involved in the process:
+- HR
+- Engineering
+- Security
+- Product
+
+#### The Project Manager
+
+The Project Manager has the responsibility of organizing materials, scheduling meetings, and generally creating a structure to operate in.
+The Project Manager is also the auditor's point of contact with the organization, funneling and prioritizing communications.
+
+As the Project Manager for Temporal's SOC 2 efforts, I cultivated a section in Notion for all of our policies, control statements, and evidence aggregation
+I established a routine way of escalating requests for evidence and decision making sessions.
```

Comment:
Making a note to not forget.
Add something about routinely broadcasting updates and statuses so everyone is in the loop.